src/Conf.ml

```diff
@@ -800,15 +800,14 @@ let profile =
   let doc = "Preset profiles." in
   let names = ["p"; "profile"] in
   let all =
-    [ ("none", Fn.id, "$(b,none) when no profile is selected.")
-    ; ( "default"
-      , (fun _ -> default)
+    [ ( "default"
+      , default
       , "$(b,default) sets each option to its default value." )
     ; ( "janestreet"
-      , (fun _ -> janestreet_profile)
+      , janestreet_profile
       , "$(b,janestreet) is the profile used at JaneStreet." ) ]
   in
-  C.choice ~names ~all ~doc ~section (fun conf f -> f conf)
+  C.choice ~names ~all ~doc ~section (fun _conf f -> f)
```

Comment:
I think that maybe I now understand why you had the `none` profile: not to actually be specified, but to be the default value as far as cmdliner is concerned. Is it currently the case that with no profile specified, processing the command line will set options using the default profile even if there are lower priority files, etc.?

Reply:
It is what I thought at first, but the choice options are actually default to None so it it is not specified it is None and not Some default, it should be OK.

Reply:
Ok, good. The only niggle is that the doc of the profile option reads like leaving it out is the same as specifying `--profile=default`, which isn't right.

Reply:
I added the possibility to add a "IGNORE" hidden value for choice options, so that this option is not shown as the default. I'm not fully satisfied with this solution but I don't have any better idea.